### PR TITLE
Adding PMIC project files.

### PR DESCRIPTION
**Pull request by kollil**
#### Rationale
Adding new project files for PMIC.

#### Related Pull Requests
* 

#### Changes
*

### CODE REVIEW

**Comment by kollil** [comment]
Josh, The changes are being tested on prime test instance. Please approve them.

**Comment by kollil** [comment]
Josh, The changes are being tested in the test instance. Please approve them to put them in uat and then in production.

Thanks

File: extscheduler/resources/queries/extscheduler/PMIC_Scheduler_Daily.sql
Comment (labkey-jeckels) [inline extscheduler/resources/queries/extscheduler/PMIC_Scheduler_Daily.sql]: Which folder(s) is this query run in? Can this container=X be removed without affecting the results? If not, we should find a different approach, as having hard-coded container IDs in queries like this makes testing on other servers that didn't come from the same database backup impossible.

File: onprc_ehr/resources/web/onprc_ehr/model/sources/PMIC_Services.js
Comment (labkey-jeckels) [inline onprc_ehr/resources/web/onprc_ehr/model/sources/PMIC_Services.js:21]: Is the '8:0' part a typo? I'm not sure what you're intending to use as the defaultValue

File: onprc_ehr/resources/schemas/dbscripts/sqlserver/onprc_ehr-20.905-20.906.sql
Comment (labkey-jeckels) [inline onprc_ehr/resources/schemas/dbscripts/sqlserver/onprc_ehr-20.905-20.906.sql]: We can't use newer SQL Server syntax in upgrade scripts as it breaks our TeamCity testing that still exercises older versions. I migrated these to use our fn_dropifexists stored procedure

**Review by labkey-jeckels — CHANGES_REQUESTED** [review]
Two comments/questions, and one note on a small adjustment I made to the SQL upgrade script

**Comment by kollil** [comment]
Josh, Could you please review the changes I made.
Thanks
Lakshmi

**Review by labkey-jeckels — APPROVED** [review]

File: onprc_ehr/resources/web/onprc_ehr/model/sources/PMIC_Services.js
Comment (kollil) [reply onprc_ehr/resources/web/onprc_ehr/model/sources/PMIC_Services.js:21]: defaulting the time to 8am

File: onprc_ehr/resources/schemas/dbscripts/sqlserver/onprc_ehr-20.905-20.906.sql
Comment (kollil) [reply onprc_ehr/resources/schemas/dbscripts/sqlserver/onprc_ehr-20.905-20.906.sql]: Thank you, the issue is fixed.

**Comment by kollil** [comment]
Josh, I am ready to merge my changes into release20.11-SNAPSHOT from 20.11_fb_PMIC. 
Before I click the Squash & Merge button, please confirm all the changes (until my recent build #132) will be included in the default branch.

Thanks
Lakshmi

**Comment by labkey-jeckels** [comment]
Yes, they will all be included. You can always review the changes being merged by going to the Files Changed tab here in the pull request.

https://github.com/LabKey/onprcEHRModules/pull/122/files

**Pull request merged**
